### ISSUE TITLE
Poorly anti-aliased Ellipses

### ISSUE THREAD

**Issue by app4soft** [LibreCAD/LibreCAD#1658]
### Expected behavior

Anti-aliasing on Ellipses should be the same quality as on Circles, Splines, PolyLines (Arcs & Lines).

### Observed behavior

Actually only on Ellipses anti-aliasing looks poorly.

![screen-2023-07-14-23-25-09](https://github.com/LibreCAD/LibreCAD/assets/22447924/1d1cf912-db90-4f15-9949-b7c95c990e08)

### Steps to reproduce or sample file

1. Enable anti-aliasing;
2. Draw Circle, Spline and Ellipse;
3. Apply line width to wider (for better visibility of issue).

### Operating System and LibreCAD version info

```
Version: 2.2.1_alpha-231-g3974b220
Compiler: GNU GCC 11.3.0
Compiled on: Jul 14 2023
Qt Version: 5.15.3
Boost Version: 1.74.0
System: Debian GNU/Linux 12 (bookworm)
```

### Related issues

- https://github.com/LibreCAD/LibreCAD/issues/778

**Comment by dxli** [comment]
Duplicate of #778

**Comment by app4soft** [comment]
Fixed with 871768d

![Screenshot_2023-07-22_16-23-40](https://github.com/LibreCAD/LibreCAD/assets/22447924/13be2d12-722c-4ea6-b2e4-5a46a5a44c33)
